src/docs/reports/2025/02/index.adoc

```diff
@@ -0,0 +1,23 @@
+= February 2025
+:icons: font
+
+== Work Package 1 - Security of the Supply Chain
+* We started a set up of a https://github.com/eclipse-apoapsis/ort-server[ORT - Server] for analyzing Apache Maven projects.
```

Comment:
If we have only one (bullet) point, we should not make it a list (start lists, as soon as there is more than one point).